### PR TITLE
Feat/delete post

### PR DESCRIPTION
**Pull request by JuanMichel16**
# Description
Method to delete post has been added and I added some code refactoring.

## What
- Method destroy in post controller added.
- Index method modified in post controller to get better performance.

See https://github.com/orgs/BrightCoders-Institute/projects/83?pane=issue&itemId=35008512 for more information.

## Why
- The users will be able to delete their posts.
- In the index method, the code made several requests in the database when it received the information.

See https://github.com/orgs/BrightCoders-Institute/projects/83?pane=issue&itemId=35008512 for more information.

Examples:

## Testing
- I created some post as test and later I deleted it.

### Code refactoring and destroy method created
image

image

### When I created some several posts
image

### After I deleted two posts
image

### CODE REVIEW

File: app/assets/stylesheets/sass/home.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
+.container {
+  display: flex;
+  height: 100%;
+  justify-content: space-between;
+  background-color: $colorBlack;
```
Comment (claunicole) [inline app/assets/stylesheets/sass/home.scss:5]: sort alphabetically

File: app/assets/stylesheets/sass/home.scss [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  overflow: hidden;
+  padding: 20px;
+  position: sticky;
+  top: 0;
+  width: 25%;
+}
+
+/* Columna de Tweets */
+.tweets-column {
+  width: 75%;
```
Comment (claunicole) [inline app/assets/stylesheets/sass/home.scss:23]: sort this too

File: app/assets/stylesheets/sass/posts.scss [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    .post-avatar {
+      border-radius: 50%;
+      height: 50px;
+      width: 50px;
+    }
+
+    .post-meta {
+      display: flex;
+      flex-direction: column;
+      color: #999;
```
Comment (claunicole) [inline app/assets/stylesheets/sass/posts.scss:29]: sort alphabetically

File: app/assets/stylesheets/sass/posts.scss [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+      position: absolute;
+      right: 15px;
+      top: 15px;
+    }
+  }
+
+  .post-content {
+    margin-bottom: 10px;
+    margin: 40px 0 40px 0;
+    display: flex;
```
Comment (claunicole) [inline app/assets/stylesheets/sass/posts.scss:49]: this too

**Review by claunicole — CHANGES_REQUESTED** [review]

**Review by sjlwa — APPROVED** [review]
Resolve conflicts

**Review by Branyoe — APPROVED** [review]

**Pull request merged**